src/components/ItemDescription.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+
+class ItemDescription extends Component {
+    constructor(props) {
+        super(props);
+    }
+    render() { 
+        return ( <div id="ItemDescription">
+            <ul>
+            {this.props.array.map((param) => 
+               <li>{param}</li>
```

Comment:
Not giving each \<li\> a key throws a "Each child in an array or iterator should have a unique key prop" warning.  I'm not sure if its a must fix, though